testsuite/setup.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
 
-# This setup script is used by the Jenkinsfile to setup miniconda and create a conda environment.
+# This setup script is used by the Jenkinsfile to setup minimamba and create a mamba environment.
 
-wget -O ${WORKSPACE}/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-bash miniconda.sh -b -p $WORKSPACE/miniconda_${NODE_NAME}
-source $WORKSPACE/miniconda_${NODE_NAME}/etc/profile.d/conda.sh
+wget -O ${WORKSPACE}/mambaforge.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
+bash mambaforge.sh -b -p $WORKSPACE/conda_${NODE_NAME}
+source $WORKSPACE/conda_${NODE_NAME}/etc/profile.d/conda.sh
+source $WORKSPACE/conda_${NODE_NAME}/etc/profile.d/mamba.sh
```

Comment:
why is both being sourced?

Reply:
https://github.com/conda-forge/miniforge#downloading-the-installer-as-part-of-a-ci-pipeline

It says there:

> Then you should create the path to conda and activate conda. Run this command:
> 
> source "${HOME}/conda/etc/profile.d/conda.sh"
> # For mamba support also run the following command
> source "${HOME}/conda/etc/profile.d/mamba.sh"

Think it basically enables both conda and mamba commands